lib/chef/knife/oci_server_create.rb

```diff
@@ -50,6 +50,14 @@ class OciServerCreate < Knife
              long: '--metadata METADATA',
              description: 'Custom metadata key/value pairs in JSON format.'
 
+      option :no_public_ip,
+             long: '--no_public_ip',
+             description: 'Do not assign a public IP address to the instance.'
+
+      option :skip_source_dest_check,
```

Comment:
Can this just be left out for now? Can you successfully complete the private IP bootstrapping scenario without this?

Reply:
Sure I'll take this part of the next code version.